e2e/browser/test-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "test-app",
-  "version": "0.1.0",
+  "version": "1.17.0",
```

Comment:
was it intentional to version the app this time, when it looks like it hasnt been bumped with previous releases?

Reply:
This is the first release since moving to npm workspaces and the app was added to the workspace as part of that; so lerna versioned it.

This doesn't really matter because the manifest sets it as a private package so a release of it will not be made.